lib/ActivityStreams.js

```diff
@@ -129,6 +132,18 @@ ActivityStreams.prototype = {
   },
 
   /**
+   * Sets up the toolbar button that links to the Timeline page.
+   */
+  _setupButton() {
+    ActionButton({
+      id: "activity-streams-link",
+      label: "Activity Streams",
```

Comment:
Spoiler alert: I have no idea where this would appear.
Are we talking about this page? https://github.com/mozilla/activity-streams/issues/106